### DISCUSSION TITLE
Autostart Rustdesk on Linux

### DISCUSSION THREAD

**Discussion by geraldurbas** [rustdesk/rustdesk#3571] [Show and tell]
Easy done by with 
sudo systemctl enable rustdesk

This is missing on Page [Docs of RustDesk](https://rustdesk.com/docs/en/) > [Manual](https://rustdesk.com/docs/en/manual/) > Linux
https://rustdesk.com/docs/en/manual/linux/

**Comment by JaCh0o** [comment]
Hmm... got installed rustdesk on arch (chimeraos) from flatpak and when I enter this command I get:
Failed to enable unit: Unit file rustdesk.service does not exists.
Any idea how to resolve this?

**Reply by vk74** [reply]
I've got the same error "Failed to enable unit: Unit file rustdesk.service does not exists." on Ubuntu. Installed via sh script

**Reply by DMSnz** [reply]
systemctl command is related to systemd init system
If your system is not systemd (Ubuntu maybe upstart init?) your commands would vary or configs to achieve this.
But I can not speak on flatpak side as I have never used them.

**Reply by vk74** [reply]
Ubuntu is using systemd, I normally use systemctl to work with other services. Anyway thanks for suggestion.

**Reply by geraldurbas** [reply]
Yes the command is for systemd or compatibles!? - lost overview;(

**Reply by ArmandRedgate** [reply]
Does anyone know the answer to how to autorun on Fedora etc. with flat pack?

I want to run my computer headless but this is impossible without this feature.

**Reply by xlionjuan** [reply]
@ArmandRedgate please install to host system if you wanna unattended access

**Reply by ArmandRedgate** [reply]
@xlionjuan thank you for your quick reply. What do you mean by this? Could you elaborate?

I have it installed both on the client (a Windows machine) and the host machine (running Fedora and installed via flatpak) but unlike on windows, when I reboot I am unable to connect until I manually start the application.

**Reply by xlionjuan** [reply]
@ArmandRedgate please install `.rpm` file.

**Reply by ArmandRedgate** [reply]
> @ArmandRedgate please install .rpm file.

~~I have, that's what I mean by flatpak.~~

EDIT:

Oh, I'm really sorry. I made a mistake. Flatpak and rpm are different things. Shows how much of a Linux noob I am.

EDIT 2: Oh god I have SELinux enabled and making it work with that looks super complicated.

**Reply by ArmandRedgate** [reply]
Ok, on Fedora, making sure I use the **.rpm** install file (you need to scroll a bit on the release page to find it) and **disabling SELinux** altogether it seems like everything works.

For anyone else in the same situation, disabling SELinux can be done by editing

```
/etc/selinux/config
```

And setting `SELINUX=disabled`

Of course, there are instructions available on rustdesk website on how to add an exception to allow rustdesk but it looks like a lot of work.

This has already been more work than I'd like to spend on installing anything (as a Windows user at least) so I personally won't be doing that.

**Comment by andreamike** [comment]
Would anyone be so kind to explain step-by-step how to do this?

**Reply by xlionjuan** [reply]
``
sudo systemctl enable --now rustdesk
``

**Reply by andreamike** [reply]
Thabk you @xlionjuan but seams to not be working. I'm using an RPi4. When I reboot I would like Rustdesk to run automatically

**Reply by xlionjuan** [reply]
You provided your hardware, but not operating system

**Reply by andreamike** [reply]
Apologies, Raspberry Pi OS (64-bit)

**Reply by xlionjuan** [reply]
It is Debian based, what is the output when you rebooted?
```
sudo systemctl status rustdesk
```

**Reply by andreamike** [reply]
@xlionjuan now it's working. Maybe I was using an outdated version of the OS/Rust. But after a clean install, it's working! Thank you
